config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -164,3 +164,33 @@ periodics:
     testgrid-alert-email: release-managers+alerts@kubernetes.io
     testgrid-dashboards: sig-release-releng-informing
     testgrid-tab-name: build-packages-rpms
+
+- interval: 6h
+  name: ci-fast-forward
+  cluster: k8s-infra-prow-build-trusted
```

Comment:
Can we submit GCB jobs to the `kubernetes-release-test` project from this cluster?